File: Java Fundamentals/DataTypesAndVariables/src/CharsToString.java
import java.util.Scanner;

public class CharsToString {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        char firstChar = scan.nextLine().charAt(0);
        char secondChar = scan.nextLine().charAt(0);
        char thirdChar = scan.nextLine().charAt(0);

        String outputString = "";
        outputString = outputString.concat(Character.toString(firstChar));
        outputString = outputString.concat(Character.toString(secondChar));
        outputString = outputString.concat(Character.toString(thirdChar));

        System.out.printf("%s",outputString);
    }
}

File: JS Front-End/JS Syntax Fundamentals/CharsToString.js
function charsToString(firstChar, secondChar, thirdChar){
    let output = firstChar;
    output += secondChar;
    output += thirdChar;

    console.log(output);
}

function reversedCharsToString(firstChar, secondChar, thirdChar){
    let output = thirdChar + " ";
    output += secondChar + " ";
    output += firstChar;

    console.log(output);
}


charsToString('a', 'b', 'c');
charsToString('%', '2', 'o');
charsToString('1', '5', 'p');


reversedCharsToString('A', 'B', 'C');
reversedCharsToString('%', '2', 'o');
reversedCharsToString('1', '5', 'p');